src/components/jellyfinApi.ts

```diff
@@ -0,0 +1,81 @@
+// this class will take over for any ajax call.
+
+import { ajax } from './fetchhelper';
+
+interface Dictionary<T> {
+    [Key: string]: T;
+}
```

Comment:
Maybe this could be moved somewhere else, or maybe there is another type better fit for a dict of string->string?

Reply:
We could move this to `.d.ts` file in `src/types`

Reply:
did :)